octo_client/client.py

```diff
@@ -25,6 +23,7 @@ def __init__(self, url: str, token: str, custom_logger: logging.Logger = None) -
         self.supplier_url_map: Dict[str, str] = {}
         self.requests_loglevel = logging.DEBUG
         self.log_responses = False
+        self.log_size_limit: Optional[int] = None
```

Comment:
We need to pass this value in `__init__`

Reply:
Oh well, 🤦‍♂️